stable/app/templates/deployment.yaml

```diff
@@ -132,6 +132,18 @@ spec:
           volumeMounts:
             - name: otelcollector-conf
               mountPath: /etc/otelcollector/
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: 13133
+            initialDelaySeconds: {{ .Values.otelcollector.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.otelcollector.readinessProbe.periodSeconds }}
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 13133
+            initialDelaySeconds: {{ .Values.otelcollector.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.otelcollector.livenessProbe.periodSeconds }}
```

Comment:
from this example
https://github.com/open-telemetry/opentelemetry-helm-charts/blob/0c72e26a6ab98f39e600e890562e80e2bf24bb40/charts/opentelemetry-operator/examples/default/rendered/deployment.yaml#L51C11-L62C30